src/main/java/sparksoniq/semantics/visitor/VisitorHelpers.java

```diff
@@ -0,0 +1,19 @@
+package sparksoniq.semantics.visitor;
+
+import sparksoniq.jsoniq.compiler.translator.expr.ExpressionOrClause;
+import sparksoniq.jsoniq.runtime.iterator.RuntimeIterator;
+import sparksoniq.semantics.StaticContext;
+
+public class VisitorHelpers {
+
+    public static RuntimeIterator generateRuntimeIterator(ExpressionOrClause expression, RuntimeIterator argument) {
```

Comment:
public static RuntimeIterator generateRuntimeIterator(ExpressionOrClause expression)

 (argument is null always)

Reply:
Done